---
title: "Terms"
previous: "Hidden/Intranet/Miriam_Annotation/RDF"
next: "Hidden/Intranet/Miriam_Annotation/CopasiML"
sitemap: false
---

{% include header.html %}
{% include navbar.html %}
{% include structure.html %}

<div class="section">
  <div class="container">
    <div class="row">
      <div class="col-xs-12">

The elements, attributes, and resource listed below are currently used in BioModels.net.<br />
<h3 name="Elements">Elements</h3>
<h4 name="a_class_wiki_external_target_blank_href_http_www_ebi_ac_uk_compneur_srv_miriam_main_mdb_section_qualifiers_rel_external_MIRIAM_Qualifiers_a_"><a target="_blank" href="http://www.ebi.ac.uk/compneur-srv/miriam-main/mdb?section=qualifiers" rel="external">MIRIAM Qualifiers</a></h4>
<table class="normal"><tr><td class="heading">Predicate URI </td><td class="heading"> User Interface </td><td class="heading"> Definition</td></tr><tr><td class="odd">bqbiol:encodes </td><td class="odd"> encodes </td><td class="odd"> The biological entity represented by the model component encodes, directly or by transitivity the subject of the referenced resource.</td></tr><tr><td class="even">bqbiol:hasPart </td><td class="even"> has&nbsp;part </td><td class="even"> The biological entity represented by the model component includes the subject of the referenced resource, either physically or logically. This relation might be used to link a complex to the description of its components.</td></tr><tr><td class="odd">bqbiol:hasVersion </td><td class="odd"> has&nbsp;version </td><td class="odd"> The subject of the referenced resource is a version or an instance of the biological entity represented by the model component.</td></tr><tr><td class="even">bqbiol:is </td><td class="even"> is </td><td class="even">  The biological entity represented by the model component is the subject of the referenced resource. This relation might be used to link a reaction to its exact counterpart in KEGG or Reactome for instance.</td></tr><tr><td class="odd">bqbiol:isDescribedBy </td><td class="odd"> reference </td><td class="odd"> The biological entity represented by the model component is described by the referenced resource. This relation should be used for instance to link a species or a parameter to the literature that describes the concentration of the species or the value of the parameter.</td></tr><tr><td class="even">bqbiol:isEncodedBy </td><td class="even"> is&nbsp;encoded&nbsp;by </td><td class="even"> The biological entity represented by the model component is encoded, directly, or by transitivity, by the subject of the referenced resource.</td></tr><tr><td class="odd">bqbiol:isHomologTo </td><td class="odd"> is&nbsp;homolog&nbsp;to </td><td class="odd"> The biological entity represented by the model component is homolog, to the subject of the referenced resource, i.e. they share a common ancestor.</td></tr><tr><td class="even">bqbiol:isPartOf </td><td class="even"> is&nbsp;part&nbsp;of </td><td class="even"> The biological entity represented by the model component is a physical or logical part of the subject of the referenced resource. This relation might be used to link a component to the description of the complex is belongs to.</td></tr><tr><td class="odd">bqbiol:isVersionOf </td><td class="odd"> is&nbsp;version&nbsp;of </td><td class="odd"> The biological entity represented by the model component is a version or an instance of the subject of the referenced resource.</td></tr></table><br />
<table class="normal"><tr><td class="heading">Predicate URI </td><td class="heading"> User Interface </td><td class="heading"> Definition</td></tr><tr><td class="odd">bqmodel:is </td><td class="odd"> is </td><td class="odd"> The modelling object represented by the model component is the subject of the referenced resource. For instance, this qualifier might be used to link the encoded model to a database of models.</td></tr><tr><td class="even">bqmodel:isDescribedBy </td><td class="even"> reference </td><td class="even"> The modelling object represented by the component of the encoded model is described by the referenced resource. This relation might be used to link a model or a kinetic law to the literature that describes this model or this kinetic law.</td></tr></table><br />
<h4 name="a_class_wiki_external_target_blank_href_http_www_w3_org_TR_vcard_rdf_rel_external_VCard_a_"><a target="_blank" href="http://www.w3.org/TR/vcard-rdf" rel="external">VCard</a></h4>
<table class="normal"><tr><td class="heading">Predicate URI </td><td class="heading"> User Interface </td><td class="heading"> Definition</td></tr><tr><td class="odd">vCard:EMAIL </td><td class="odd"> Email </td><td class="odd"></td></tr><tr><td class="even">vCard:Family </td><td class="even"> Family Name </td><td class="even"></td></tr><tr><td class="odd">vCard:Given </td><td class="odd"> Given Name </td><td class="odd"></td></tr><tr><td class="even">vCard:N </td><td class="even"> Full Name </td><td class="even"></td></tr><tr><td class="odd">vCard:ORG </td><td class="odd"> Organization </td><td class="odd"></td></tr><tr><td class="even">vCard:Orgname </td><td class="even"> Organization Name </td><td class="even"></td></tr></table><br />
<h4 name="a_class_wiki_external_target_blank_href_http_dublincore_org_rel_external_Dublin_Core_a_"><a target="_blank" href="http://dublincore.org/" rel="external">Dublin Core</a></h4>
<table class="normal"><tr><td class="heading">Predicate URI </td><td class="heading"> User Interface </td><td class="heading"> Definition</td></tr><tr><td class="odd">dc:creator </td><td class="odd"> creator </td><td class="odd"> An entity primarily responsible for making the resource.</td></tr><tr><td class="even">dc:relation </td><td class="even"> relation </td><td class="even"> A related resource.</td></tr><tr><td class="odd">dcterms:W3CDTF </td><td class="odd"> encoding for dates (hidden) </td><td class="odd"> The set of dates and times constructed according to the <a target="_blank" href="http://www.w3.org/TR/NOTE-datetime" rel="external">W3C Date and Time Formats Specification</a>.</td></tr><tr><td class="even">dcterms:created </td><td class="even"> date created </td><td class="even"> Date of creation of the resource.</td></tr><tr><td class="odd">dcterms:modified </td><td class="odd"> date modified </td><td class="odd"> Date on which the resource was changed.</td></tr><tr><td class="even">dcterms:hasPart </td><td class="even"> has part </td><td class="even"> A related resource that is included either physically or logically in the described resource.</td></tr><tr><td class="odd">dcterms:hasVersion </td><td class="odd">  has version </td><td class="odd"> A related resource that is a version, edition, or adaptation of the described resource.</td></tr></table><br />
<h4 name="a_class_wiki_external_target_blank_href_http_www_w3_org_TR_rdf_syntax_grammar_rel_external_RDF_XML_Syntax_a_"><a target="_blank" href="http://www.w3.org/TR/rdf-syntax-grammar/" rel="external">RDF/XML Syntax</a></h4>
<table class="normal"><tr><td class="heading">Element Name </td><td class="heading"> User Interface</td></tr><tr><td class="odd">rdf:Bag </td><td class="odd"> hidden</td></tr><tr><td class="even">rdf:Description </td><td class="even"> hidden</td></tr><tr><td class="odd">rdf:RDF </td><td class="odd"> hidden</td></tr><tr><td class="even">rdf:li </td><td class="even"> continuous numbering sequence (1., 2., ...)</td></tr><tr><td class="odd">rdf:_# </td><td class="odd"> #. (where # is a positive integer)</td></tr></table><br />
<table class="normal"><tr><td class="heading">Attribute Name </td><td class="heading"> User Interface</td></tr><tr><td class="odd">rdf:about </td><td class="odd"> show attribute value as subject</td></tr><tr><td class="even">rdf:parseType </td><td class="even"> hidden</td></tr><tr><td class="odd">rdf:resource </td><td class="odd"> show interpreted attribute value as object</td></tr></table><br />
<h3 name="a_class_wiki_external_target_blank_href_http_www_ebi_ac_uk_compneur_srv_miriam_main_mdb_section_browse_rel_external_MIRIAM_Resources_a_"><a target="_blank" href="http://www.ebi.ac.uk/compneur-srv/miriam-main/mdb?section=browse" rel="external">MIRIAM Resources</a></h3>
<h4 name="URI_Bases">URI Bases</h4>
<table class="normal"><tr><td class="heading">Name </td><td class="heading"> URI Base </td><td class="heading"> Description</td></tr><tr><td class="odd"><a target="_blank" href="http://www.ebi.ac.uk/compneur-srv/miriam-main/mdb?section=browse&amp;data=MIR:00000036" rel="external">ArrayExpress</a> </td><td class="odd">   http://arrayexpress.org/   </td><td class="odd"> ArrayExpress is a public repository for microarray data, which is aimed at storing MIAME-compliant data in accordance with Microarray Gene Expression Data (MGED) recommendations.</td></tr><tr><td class="even"><a target="_blank" href="http://www.ebi.ac.uk/compneur-srv/miriam-main/mdb?section=browse&amp;data=MIR:00000035" rel="external">arXiv</a> </td><td class="even">   http://arxiv.org/   </td><td class="even"> arXiv is an e-print service in the fields of physics, mathematics, non-linear science, computer science, and quantitative biology.</td></tr><tr><td class="odd"><a target="_blank" href="http://www.ebi.ac.uk/compneur-srv/miriam-main/mdb?section=browse&amp;data=MIR:00000001" rel="external">BIND</a> </td><td class="odd">   http://www.bind.ca/   </td><td class="odd"> BIND is a database of protein-protein interactions. This data-resource is not open-access.</td></tr><tr><td class="even"><a target="_blank" href="http://www.ebi.ac.uk/compneur-srv/miriam-main/mdb?section=browse&amp;data=MIR:00000007" rel="external">BioModels Database</a> </td><td class="even">   http://www.ebi.ac.uk/biomodels/   </td><td class="even"> BioModels Database is a data resource that allows biologists to store, search and retrieve published mathematical models of biological interests.</td></tr><tr><td class="odd"><a target="_blank" href="http://www.ebi.ac.uk/compneur-srv/miriam-main/mdb?section=browse&amp;data=MIR:00000002" rel="external">ChEBI</a> </td><td class="odd">   http://www.ebi.ac.uk/chebi/   </td><td class="odd"> Chemical Entities of Biological Interest (ChEBI) is a freely available dictionary of molecular entities focused on 'small' chemical compounds.</td></tr><tr><td class="even"><a target="_blank" href="http://www.ebi.ac.uk/compneur-srv/miriam-main/mdb?section=browse&amp;data=MIR:00000021" rel="external">CluSTr</a> </td><td class="even">   http://www.ebi.ac.uk/clustr/   </td><td class="even"> The CluSTr database offers an automatic classification of UniProt Knowledgebase and IPI proteins into groups of related proteins. The clustering is based on analysis of all pairwise comparisons (Smith-Waterman) between protein sequences.</td></tr><tr><td class="odd"><a target="_blank" href="http://www.ebi.ac.uk/compneur-srv/miriam-main/mdb?section=browse&amp;data=MIR:00000019" rel="external">DOI</a> </td><td class="odd">   http://www.doi.org/   </td><td class="odd"> The Digital Object Identifier System is for identifying content objects in the digital environment.</td></tr><tr><td class="even"><a target="_blank" href="http://www.ebi.ac.uk/compneur-srv/miriam-main/mdb?section=browse&amp;data=MIR:00000003" rel="external">Ensembl</a> </td><td class="even">   http://www.ensembl.org/   </td><td class="even"> Ensembl is a joint project between EMBL - EBI and the Sanger Institute  to develop a software system which produces and maintains automatic annotation on selected eukaryotic genomes.</td></tr><tr><td class="odd"><a target="_blank" href="http://www.ebi.ac.uk/compneur-srv/miriam-main/mdb?section=browse&amp;data=MIR:00000004" rel="external">Enzyme Nomenclature</a> </td><td class="odd">   http://www.ec-code.org/   </td><td class="odd"> The Enzyme Classification contains the recommendations of the Nomenclature Committee of the International Union of Biochemistry and Molecular Biology on the nomenclature and classification of enzyme-catalysed reactions.</td></tr><tr><td class="even"><a target="_blank" href="http://www.ebi.ac.uk/compneur-srv/miriam-main/mdb?section=browse&amp;data=MIR:00000030" rel="external">FlyBase</a> </td><td class="even">   http://www.flybase.org/   </td><td class="even"> FlyBase is the database of the Drosophila Genome Projects and of associated literature.</td></tr><tr><td class="odd"><a target="_blank" href="http://www.ebi.ac.uk/compneur-srv/miriam-main/mdb?section=browse&amp;data=MIR:00000022" rel="external">Gene Ontology</a> </td><td class="odd">   http://www.geneontology.org/   </td><td class="odd"> The Gene Ontology project provides a controlled vocabulary to describe gene and gene product attributes in any organism.</td></tr><tr><td class="even"><a target="_blank" href="http://www.ebi.ac.uk/compneur-srv/miriam-main/mdb?section=browse&amp;data=MIR:00000009" rel="external">ICD</a> </td><td class="even">   http://www.who.int/classifications/icd/   </td><td class="even"> The International Classification of Diseases is the international standard diagnostic classification for all general epidemiological and many health management purposes.</td></tr><tr><td class="odd"><a target="_blank" href="http://www.ebi.ac.uk/compneur-srv/miriam-main/mdb?section=browse&amp;data=MIR:00000010" rel="external">IntAct</a> </td><td class="odd">   http://www.ebi.ac.uk/intact/   </td><td class="odd"> IntAct provides a freely available, open source database system and analysis tools for protein interaction data.</td></tr><tr><td class="even"><a target="_blank" href="http://www.ebi.ac.uk/compneur-srv/miriam-main/mdb?section=browse&amp;data=MIR:00000011" rel="external">InterPro</a> </td><td class="even">   http://www.ebi.ac.uk/interpro/   </td><td class="even"> InterPro is a database of protein families, domains and functional sites in which identifiable features found in known proteins can be applied to unknown protein sequences.</td></tr><tr><td class="odd"><a target="_blank" href="http://www.ebi.ac.uk/compneur-srv/miriam-main/mdb?section=browse&amp;data=MIR:00000013" rel="external">KEGG Compound</a> </td><td class="odd">   http://www.genome.jp/kegg/compound/   </td><td class="odd"> KEGG compound contains our knowledge on the universe of chemical substances that are relevant to life.</td></tr><tr><td class="even"><a target="_blank" href="http://www.ebi.ac.uk/compneur-srv/miriam-main/mdb?section=browse&amp;data=MIR:00000025" rel="external">KEGG Drug</a> </td><td class="even">   http://www.genome.jp/kegg/drug/   </td><td class="even"> KEGG DRUG contains chemical structures of drugs and additional information such as therapeutic categories and target molecules.</td></tr><tr><td class="odd"><a target="_blank" href="http://www.ebi.ac.uk/compneur-srv/miriam-main/mdb?section=browse&amp;data=MIR:00000026" rel="external">KEGG Glycan</a> </td><td class="odd">   http://www.genome.jp/kegg/glycan/   </td><td class="odd"> KEGG GLYCAN, a part of the KEGG LIGAND database, is a collection of experimentally determined glycan structures. It contains all unique structures taken from CarbBank, structures entered from recent publications, and structures present in KEGG pathways.</td></tr><tr><td class="even"><a target="_blank" href="http://www.ebi.ac.uk/compneur-srv/miriam-main/mdb?section=browse&amp;data=MIR:00000012" rel="external">KEGG Pathway</a> </td><td class="even">   http://www.genome.jp/kegg/pathway/   </td><td class="even"> KEGG PATHWAY is a collection of manually drawn pathway maps representing our knowledge on the molecular interaction and reaction networks.</td></tr><tr><td class="odd"><a target="_blank" href="http://www.ebi.ac.uk/compneur-srv/miriam-main/mdb?section=browse&amp;data=MIR:00000014" rel="external">KEGG Reaction</a> </td><td class="odd">   http://www.genome.jp/kegg/reaction/   </td><td class="odd"> KEGG reaction contains our knowledge on the universe of reactions that are relevant to life.</td></tr><tr><td class="even"><a target="_blank" href="http://www.ebi.ac.uk/compneur-srv/miriam-main/mdb?section=browse&amp;data=MIR:00000008" rel="external">MIRIAM Resources</a> </td><td class="even">   http://biomodels.net/MIRIAM/   </td><td class="even"> MIRIAM Resources is an online resource created to catalogue the data-types (Gene Ontology, Taxonomy or PubMed are some examples), their URIs and the corresponding physical URLs, whether these are controlled vocabularies or databases.</td></tr><tr><td class="odd"><a target="_blank" href="http://www.ebi.ac.uk/compneur-srv/miriam-main/mdb?section=browse&amp;data=MIR:00000037" rel="external">Mouse Genome Database</a> </td><td class="odd">   http://mousegenome.org/   </td><td class="odd"> The Mouse Genome Database (MGD) project includes data on gene characterization, nomenclature, mapping, gene homologies among mammals, sequence links, phenotypes, allelic variants and mutants, and strain data.</td></tr><tr><td class="even"><a target="_blank" href="http://www.ebi.ac.uk/compneur-srv/miriam-main/mdb?section=browse&amp;data=MIR:00000029" rel="external">Nucleotide Sequence Database</a> </td><td class="even">   http://www.insdc.org/   </td><td class="even"> The International Nucleotide Sequence Database Collaboration (INSDC) consists of a joint effort to collect and disseminate databases containing DNA and RNA sequences.</td></tr><tr><td class="odd"><a target="_blank" href="http://www.ebi.ac.uk/compneur-srv/miriam-main/mdb?section=browse&amp;data=MIR:00000016" rel="external">OMIM</a> </td><td class="odd">   http://www.ncbi.nlm.nih.gov/OMIM/   </td><td class="odd"> Online Mendelian Inheritance in Man is a catalog of human genes and genetic disorders.</td></tr><tr><td class="even"><a target="_blank" href="http://www.ebi.ac.uk/compneur-srv/miriam-main/mdb?section=browse&amp;data=MIR:00000028" rel="external">Pfam</a> </td><td class="even">   http://pfam.org/   </td><td class="even"> The Pfam database contains information about protein domains and families. For each entry a protein sequence alignment and a Hidden Markov Model is stored.</td></tr><tr><td class="odd"><a target="_blank" href="http://www.ebi.ac.uk/compneur-srv/miriam-main/mdb?section=browse&amp;data=MIR:00000017" rel="external">PIRSF</a> </td><td class="odd">   http://pir.georgetown.edu/pirsf/   </td><td class="odd"> The PIR SuperFamily concept is being used as a guiding principle to provide comprehensive and non-overlapping clustering of UniProtKB sequences into a hierarchical order to reflect their evolutionary relationships.</td></tr><tr><td class="even"><a target="_blank" href="http://www.ebi.ac.uk/compneur-srv/miriam-main/mdb?section=browse&amp;data=MIR:00000032" rel="external">PROSITE</a> </td><td class="even">   http://www.prosite.org/   </td><td class="even"> PROSITE consists of documentation entries describing protein domains, families and functional sites as well as associated patterns and profiles to identify them.</td></tr><tr><td class="odd"><a target="_blank" href="http://www.ebi.ac.uk/compneur-srv/miriam-main/mdb?section=browse&amp;data=MIR:00000020" rel="external">Protein Data Bank</a> </td><td class="odd">   http://www.pdb.org/   </td><td class="odd"> The Protein Data Bank is the single worldwide archive of structural data of biological macromolecules.</td></tr><tr><td class="even"><a target="_blank" href="http://www.ebi.ac.uk/compneur-srv/miriam-main/mdb?section=browse&amp;data=MIR:00000034" rel="external">PubChem-compound</a> </td><td class="even">   http://www.pubchem.gov/compound/   </td><td class="even"> PubChem provides information on the biological activities of small molecules. It is a component of NIH's Molecular Libraries Roadmap Initiative. PubChem Substance archives chemical structures. records.</td></tr><tr><td class="odd"><a target="_blank" href="http://www.ebi.ac.uk/compneur-srv/miriam-main/mdb?section=browse&amp;data=MIR:00000033" rel="external">PubChem-substance</a> </td><td class="odd">   http://www.pubchem.gov/substance/   </td><td class="odd"> PubChem provides information on the biological activities of small molecules. It is a component of NIH's Molecular Libraries Roadmap Initiative. PubChem Substance archives chemical substance records.</td></tr><tr><td class="even"><a target="_blank" href="http://www.ebi.ac.uk/compneur-srv/miriam-main/mdb?section=browse&amp;data=MIR:00000015" rel="external">PubMed</a> </td><td class="even">   http://www.pubmed.gov/   </td><td class="even"> PubMed is a service of the U.S. National Library of Medicine that includes citations from MEDLINE and other life science journals for biomedical articles back to the 1950s.</td></tr><tr><td class="odd"><a target="_blank" href="http://www.ebi.ac.uk/compneur-srv/miriam-main/mdb?section=browse&amp;data=MIR:00000018" rel="external">Reactome</a> </td><td class="odd">   http://www.reactome.org/   </td><td class="odd"> The Reactome project is a collaboration to develop a curated resource of core pathways and reactions in human biology.</td></tr><tr><td class="even"><a target="_blank" href="http://www.ebi.ac.uk/compneur-srv/miriam-main/mdb?section=browse&amp;data=MIR:00000038" rel="external">SABIO-RK Reaction</a> </td><td class="even">   http://sabio.villa-bosch.de/reaction/   </td><td class="even"> SABIO-RK is a relational database system that contains information about biochemical reactions, their kinetic equations with their parameters, and the experimental conditions under which these parameters were measured.</td></tr><tr><td class="odd"><a target="_blank" href="http://www.ebi.ac.uk/compneur-srv/miriam-main/mdb?section=browse&amp;data=MIR:00000023" rel="external">Saccharomyces genome database</a> </td><td class="odd">   http://www.yeastgenome.org/   </td><td class="odd"> The Saccharomyces Genome Database (SGD) project collects information and maintains a database of the molecular biology of the yeast Saccharomyces cerevisiae.</td></tr><tr><td class="even"><a target="_blank" href="http://www.ebi.ac.uk/compneur-srv/miriam-main/mdb?section=browse&amp;data=MIR:00000024" rel="external">Systems Biology Ontology</a> </td><td class="even">   http://biomodels.net/SBO/   </td><td class="even"> The goal of the Systems Biology Ontology is to develop controlled vocabularies and ontologies tailored specifically for the kinds of problems being faced in Systems Biology, especially in the context of computational modeling. SBO is a project of the BioModels.net effort.</td></tr><tr><td class="odd"><a target="_blank" href="http://www.ebi.ac.uk/compneur-srv/miriam-main/mdb?section=browse&amp;data=MIR:00000006" rel="external">Taxonomy</a> </td><td class="odd">   http://www.taxonomy.org/   </td><td class="odd"> The taxonomy contains the relationships between all living forms for which nucleic acid or protein sequence have been determined.</td></tr><tr><td class="even"><a target="_blank" href="http://www.ebi.ac.uk/compneur-srv/miriam-main/mdb?section=browse&amp;data=MIR:00000005" rel="external">UniProt</a> </td><td class="even">   urn:lsid:uniprot.org:uniprot   </td><td class="even"> UniProt (Universal Protein Resource) is the world's most comprehensive catalog of information on proteins. It is a central repository of protein sequence and function created by joining the information contained in Swiss-Prot, TrEMBL, and PIR.</td></tr><tr><td class="odd"><a target="_blank" href="http://www.ebi.ac.uk/compneur-srv/miriam-main/mdb?section=browse&amp;data=MIR:00000027" rel="external">WormBase</a> </td><td class="odd">   http://www.wormbase.org/   </td><td class="odd"> WormBase is an online bioinformatics database of the biology and genome of the model organism Caenorhabditis elegans and related nematodes. It is used by the C. elegans research community both as an information resource and as a mode to publish and distribute their results.</td></tr><tr><td class="even"><a target="_blank" href="http://www.ebi.ac.uk/compneur-srv/miriam-main/mdb?section=browse&amp;data=MIR:00000031" rel="external">Wormpep</a> </td><td class="even">   http://www.wormpep.org/   </td><td class="even"> Wormpep contains the predicted proteins from the Caenorhabditis elegans genome sequencing project.</td></tr></table><br />
BioModels currently uses:<br />
<ul><li>http://pir.georgetown.edu/pirsf/
</li><li>http://www.bind.ca/
</li><li>http://www.doi.org/
</li><li>http://www.ebi.ac.uk/IntEnz/
</li><li>http://www.ebi.ac.uk/biomodels/
</li><li>http://www.ebi.ac.uk/chebi/
</li><li>http://www.ebi.ac.uk/intact/
</li><li>http://www.ebi.ac.uk/interpro/
</li><li>http://www.ec-code.org/
</li><li>http://www.ensembl.org/
</li><li>http://www.geneontology.org/
</li><li>http://www.genome.jp/kegg/compound/
</li><li>http://www.genome.jp/kegg/pathway/
</li><li>http://www.genome.jp/kegg/reaction/
</li><li>http://www.ncbi.nlm.nih.gov/OMIM/
</li><li>http://www.ncbi.nlm.nih.gov/PubMed/
</li><li>http://www.ncbi.nlm.nih.gov/Taxonomy/
</li><li>http://www.pubmed.gov/
</li><li>http://www.reactome.org/
</li><li>http://www.taxonomy.org/
</li><li>http://www.uniprot.org/
</li><li>http://www.who.int/classifications/icd/
</li></ul>
<h4 name="ur_x_l">ur<x>l</h4>
<ul><li>http://www.icsb2001.org/Papers/08_Kongas_Paper.pdf
</li><li>http://www.genome.ad.jp/dbget-bin/show_pathway?map00590+1.14.15.3
</li></ul>
<br />

      </div>
    </div>
  </div>
</div>

<!-- FOOTER  -->
{% include footer.html %}
